### PR TITLE
Re-instate Julia 1.6 support

### LINKED ISSUE #29: Julia 1.6 support
Is there a good reason not to support Julia 1.6?

x-ref: https://github.com/SciML/Optimization.jl/pull/274#issuecomment-1148170500

### PR DESCRIPTION
**Pull request by odow**
Let's see if this works.

Closes #29

### CODE REVIEW

**Comment by odow** [comment]
Working for me locally:
image

File: Manifest.toml [hunk 1]
```diff
@@ -1,219 +0,0 @@
-# This file is machine-generated - editing it directly is not advised
```
Comment (odow) [inline Manifest.toml:1]: Perhaps your earlier problem was checking `Manifest.toml` into the repository? For packages you need only `Project.toml`.

**Comment by odow** [comment]
Sorry to be a pain, I pushed a new commit 😆 Seems like the tests are passing though which is good. Thanks for your help.

File: Manifest.toml (hunk 1 above)
Comment (sumiya11) [reply Manifest.toml:1]: Oh, didn't know that 😅

**Comment by sumiya11** [comment]
No worries; tests also pass locally for me on 1.6, maybe `Manifest.toml` was a problem 😄

**Pull request merged**